### DISCUSSION TITLE
browser loading old update of hub (re: fog)

### DISCUSSION THREAD

**Discussion by marbusbun** [mozilla/hubs#4066] [General Questions & Discussion] [answered]
hey hey! Needing some help/ guidance and wondering if anyone has had the same problem.

I'm working on a project - accessed at this link -  https://hubs.mozilla.com/VCBH2rP/recollecting-futures

Finding that sometimes the internet browser is loading an older copy of this project (shows an old spawn point, and no fog). 

when I refresh my browser page, and re-enter the hub, then it's loading the newest edition (with correct spawn point and with fog).

Is this a mozilla hubs server lag? Anyone know anything about a problem like this? THANKS in advance!!

**Comment by johnshaughnessy** [comment, marked as answer]
Hi @marbusbun . It sounds like your browser might be holding onto a cached copy of a file because it did not know if changed. Next time you are working on this, try opening the dev tools UI in your browser and checking "disable cache" in the network tab to see if that's the problem.

![image](https://user-images.githubusercontent.com/4072106/113202391-8b154300-921f-11eb-904a-8adae92366b8.png)
![image](https://user-images.githubusercontent.com/4072106/113202405-8e103380-921f-11eb-9e2d-572585dc47db.png)

If this does not fix your issue, unmark this response as the answer and let me know if you still have the same problem.
